### PR TITLE
CA currency migration.

### PR DESCRIPTION
**Pull request by norisalsaadie**
components/ca-currency.css - > direct copy
components/ca-currency.js -> ES6 + AMD module
examples/ca-currency.html -> copy
examples/index.html -> appending currency to demo list
test/ca-currency-test.html -> migrated and fixed.

### CODE REVIEW

File: test/ca-currency-test.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                expect(currency.readonly).to.equal(true);
+            });
+
+            test('should create the relevant structure', (done) => {
+                expect(currency.value).to.equal("123.45");
+
+                setTimeout(() => {
+                    const input = document.querySelector('ca-currency input');
+                    expect(input).to.be.defined;
+                    debugger;
```
Comment (kevmarchant) [inline test/ca-currency-test.html]: debugger statement left in for comedy value.

Comment (norisalsaadie) [inline test/ca-currency-test.html]: 💃 this was no joke, by an accident of epic scale.

File: test/ca-currency-test.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>CA DIALOG | Web Lab Common UI</title>
```
Comment (joeburton) [inline test/ca-currency-test.html]: wrong title

File: test/ca-currency-test.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    <link rel="stylesheet" type="text/css" href="/components/ca-currency.css">
+  </head>
+  <body>
+    <test-fixture id="currency">
+      <template>
+        <ca-currency name="Name" symbol="$" value="123.45" readonly="true"></ca-currency>
+      </template>
+    </test-fixture>
+    <script>
+        suite('<ca-dialog>', () => {
```
Comment (joeburton) [inline test/ca-currency-test.html]: should be ca-currency

**Pull request merged**